doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -344,7 +344,12 @@ Keys samples
 Matter samples
 --------------
 
-* Changed Bluetooth Low Energy variant of the Soft Device Controller (SDC) to use the Peripheral-only role in all Matter samples.
+* Added the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILE_NAME` Kconfig option to configure the ZAP file name used by the sample.
+
+* Changed:
+
+  * Bluetooth Low Energy variant of the Soft Device Controller (SDC) to use the Peripheral-only role in all Matter samples.
+  * API of the ``ncs_configure_data_model`` cmake method that does not use ``ZAP_FILE`` argument anymore, but creates path to ZAP file based on :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILES_PATH` and :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILE_NAME` Kconfig options.
```

Comment:
```suggestion
* Updated:

  * The Bluetooth Low Energy variant of the Soft Device Controller (SDC) to use the Peripheral-only role in all Matter samples.
  * API of the ``ncs_configure_data_model`` cmake method that does not use ``ZAP_FILE`` argument anymore, but creates path to ZAP file based on :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILES_PATH` and :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILE_NAME` Kconfig options.
```
Sorry, I tend to forget that we do not use "heading" Changed. Instead, please use Updated.